tests/test_converter.py

```diff
@@ -0,0 +1,102 @@
+"""Unit test for converter."""
+
+from pathlib import Path
+import numpy as np
+import pytest
+import xarray as xr
+from zampy.datasets import EthCanopyHeight
+from zampy.datasets import converter
+from zampy.datasets.eth_canopy_height import parse_tiff_file
+
+
+path_dummy_data = Path(__file__).resolve().parent / "test_data" / "eth-canopy-height"
+
+# ruff: noqa: B018
+
+
+def test_check_convention_not_support():
+    convention = "fake_convention"
+    with pytest.raises(ValueError, match="not supported"):
+        converter.check_convention(convention)
+
+
+def test_check_convention_not_exist():
+    convention = Path("fake_path")
+    with pytest.raises(FileNotFoundError, match="could not be found"):
+        converter.check_convention(convention)
+
+
+def test_convert_var():
+    """Test _convert_var function."""
+    ds = parse_tiff_file(
+        path_dummy_data / "ETH_GlobalCanopyHeight_10m_2020_N51E003_Map.tif",
+    )
+    ds_convert = converter._convert_var(ds, "height_of_vegetation", "decimeter")
+
+    assert np.allclose(
+        ds_convert["height_of_vegetation"].values,
+        ds["height_of_vegetation"].values * 10.0,
+        equal_nan=True,
+    )
+
+
+def test_convert_var_name():
+    """Test convert function.
+
+    In this test, no conversion is performed. Only the variable name is updated.
+    """
+    ds = parse_tiff_file(
+        path_dummy_data / "ETH_GlobalCanopyHeight_10m_2020_N51E003_Map.tif",
+    )
+    ds_convert = converter.convert(
+        data=ds, dataset=EthCanopyHeight(), convention="ALMA"
+    )
+
+    assert list(ds_convert.data_vars)[0] == "Hveg"
```

Comment:
Changes according to a convention can include three things: change in variable names, changes in units, changes in coordinates. Only unit conversion is considered in the current implementation, see [here](https://github.com/EcoExtreML/zampy/blob/54439a0e923da7a1ddeac813e83bc8e7088fa1dc/src/zampy/datasets/converter.py#L64-L75). I think if the name of the variable is changed according to the convention, `converted` should be `True`, right?

Reply:
Good point. I think it makes sense. Now either the name of variable or the unit conversion is triggered, the `converted` flag will be `True`.